migration/migrator/migrations/course/20200529001923_cgi_tmp_folder.py

```diff
@@ -0,0 +1,22 @@
+import os
+import grp
+from pathlib import Path
+
+
+def up(config, database, semester, course):
+    tmp_cgi_dir = Path(config.submitty['submitty_data_dir'], 'tmp', 'cgi')
+
+    # create the directories
+    os.makedirs(str(tmp_cgi_dir), exist_ok=True)
+
+    php_user = config.submitty_users['php_user']
+    daemoncgi_group = config.submitty_users['daemoncgi_group']
+
+    # set the owner/group/permissions
+    os.system("chown -R "+php_user+":"+daemoncgi_group+" "+str(tmp_cgi_dir))
+    os.system("chmod -R u+rwx  "+str(tmp_cgi_dir))
+    os.system("chmod -R g+rwxs "+str(tmp_cgi_dir))
+    os.system("chmod -R o-rwx  "+str(tmp_cgi_dir))
```

Comment:
If doing these with pure python is not easily done, please use `subprocess.check_output` instead. We should avoid `os.system` as much as possible as if it fails, it will not propagate properly within the migrator.